genologics/lims.py

```diff
@@ -585,6 +586,11 @@ def tostring(self, etree):
         self.write(outfile, etree)
         return outfile.getvalue()
 
+    def is_equal(self, entity1, entity2, exclude_tag=None):
+        c1 = ComparableXml(ElementTree.tostring(entity1.root), exclude_tag=exclude_tag)
+        c2 = ComparableXml(ElementTree.tostring(entity2.root), exclude_tag=exclude_tag)
+        return c1.tostring() == c2.tostring()
```

Comment:
As a developer using the class `ComparableXml` I think I would expect being able to compare them like this:

```
c1 == c2
```

i.e. without having to call `tostring`.